### ISSUE TITLE
XRI 2.3 Upgrade

### ISSUE THREAD

**Issue by Zee2** [microsoft/MixedRealityToolkit-Unity#11282]
This ticket tracks upgrade to XRI 2.3, along with the XR coreutils upgrade

- [ ] Evaluate new input simulator, how much of our own input simulator we can remove
- [ ] Evaluate Transformers and how they will influence our constraints/solvers story
- [ ] Evaluate the new datum feature in XR Coreutils, possibly use it for config on BC, UX?
- [ ] Tests pass
- [ ] Check in with internal partners before bumping XRI minver

**Comment by RogPodge** [comment]
2.3 contains some API improvements for key features such as implementing custom reticles for bound handles. So long as our existing implementations don't break, I'd like to move to 2.3 even without completing the other evaluations listed on this task.

**Comment by Zee2** [comment]
@RogPodge I agree mostly, except for the most egregious issues here (i.e., two completely duplicate `PokeInteractor`s., etc) I will be bringing up as many of these as possible in our meeting tomorrow.

**Comment by AMollis** [comment]
See #11298 for draft PR

**Comment by shaynie** [comment]
Need to go through the investigations and recommendations listed above and create feature requests.

**Comment by shaynie** [comment]
Updated to XRI 2.3 with https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11452
